### PR TITLE
Add custom node suffix

### PR DESCRIPTION
**Pull request by malachico**
### 🔦 Summary
...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/MultiSelect/InputWithTags.js [hunk 1]
```diff
@@ -241,6 +246,8 @@ InputWithTags.propTypes = {
   mode: PropTypes.oneOf(['select']),
   delimiters: PropTypes.array,
   width: PropTypes.string,
+  customSuffixNode: PropTypes.node,
+  isDropdownOpen: PropTypes.func,
```
[the remark this answers is not in the document]
Comment (malachico) [reply src/MultiSelect/InputWithTags.js]: sounds good

[the remark this answers is not in the document]
Comment (argshook) [reply src/MultiSelect/InputWithTags.js]: why is `isDropdownOpen` a function?

[the remark this answers is not in the document]
Comment (nitayneeman) [reply src/MultiSelect/InputWithTags.js]: He's right - a boolean is enough and actually even clearer.

For example, the usage in Multiselect:
```javascript
      inputElement: (
        <InputWithTags
          onReorder={this.props.onReorder}
          maxNumRows={this.props.maxNumRows}
          mode={this.props.mode}
          customSuffix={this.props.customSuffix}
          hideCustomSuffix={this.isDropdownLayoutVisible()}
        />
```

And then:
```javascript
        {this.props.customSuffix && !this.props.hideCustomSuffix && (
          <span data-hook="custom-suffix" className={styles.customSuffix}>
            {this.props.customSuffix}
          </span>
        )}
```

**Comment by nitayneeman** [comment]
@malachico Seems like there are some conflicts in Multiselect spec

**Pull request merged**